modules/sharing-registry/sharing-registry-server/src/main/java/org/apache/airavata/sharing/registry/server/SharingRegistryServerHandler.java

```diff
@@ -277,7 +277,9 @@ public String createGroup(UserGroup group) throws SharingRegistryException, TExc
             group.setUpdatedTime(System.currentTimeMillis());
             (new UserGroupRepository()).create(group);
 
-            addUsersToGroup(group.domainId, Arrays.asList(group.ownerId), group.groupId);
+            if(group.ownerId != null)
```

Comment:
Sneha, instead of having no owner, let's make the owner an argument to the migration script like you did in the other pull request.

Reply:
Sure.